ceremonies/src/lib.rs

```diff
@@ -310,22 +321,42 @@ decl_module! {
             <AttestationRegistry<T>>::insert((cid, cindex), &idx, &verified_attestation_accounts);
             <AttestationIndex<T>>::insert((cid, cindex), &sender, &idx);
             <MeetupParticipantCountVote<T>>::insert((cid, cindex), &sender, &claim_n_participants);
-            debug::debug!(target: LOG, 
-                "sucessfully registered {} attestations for {:?}", 
+            debug::debug!(target: LOG,
+                "sucessfully registered {} attestations for {:?}",
                 verified_attestation_accounts.len(), sender);
             Ok(())
         }
-    }
-}
 
-decl_event!(
-    pub enum Event<T>
-    where
-        AccountId = <T as frame_system::Trait>::AccountId,
-    {
-        ParticipantRegistered(AccountId),
+        #[weight = 10_000]
+        pub fn endorse_newcomer(origin, cid: CurrencyIdentifier, newby: T::AccountId) -> DispatchResult {
+            debug::RuntimeLogger::init();
+            let sender = ensure_signed(origin)?;
+
+            // this is done to simplify the logic as a trade-off for UX, subject to change?
```

Comment:
@brenzi shall we change that such that endorsements in the assigning/attesting phase will be effective for the ceremony with `currency_ceremony_index + 1`?

Reply:
that makes sense. it should be possible to endorse anytime, irrespective of the phase.

Reply:
What do we do if a newcomer misses the ceremony? should the endorsement be void then? I think we could live with that for now. the bootsrapper could re-endorse

Reply:
This is what I thought. Someone who cannot attend after being endorsed is actually not very reputable?

But we could migrate them to (cid, cindex + 1) but then there is a chance that some endorsees might never turn up and we migrate them forever. Even if we solve it differently, we should have some kind of mechanism to track endorsees who never turn up to free unused storage, shouldn't we?

Reply:
lets keep it as a single chance. the endorser can endorse again, but that'll cost another credit